pandas/core/interchange/dataframe.py

```diff
@@ -87,7 +87,7 @@ def select_columns(self, indices: Sequence[int]) -> PandasDataFrameXchg:
             self._df.iloc[:, indices], self._nan_as_null, self._allow_copy
         )
 
-    def select_columns_by_name(self, names: Sequence[str]) -> PandasDataFrameXchg:
+    def select_columns_by_name(self, names: list[str]) -> PandasDataFrameXchg:  # type: ignore[override]  # noqa: E501
```

Comment:
This is pretty interesting that you have to do the override.  In some sense, the method `DataFrameXchg.select_columns_by_name()` is wrong when it says `names` is `Sequence[str]`, because probably the people who wrote that aren't aware that a pure `str` will match.

Reply:
Yeah this might be an oversight on the original spec. I opened an issue regarding this https://github.com/data-apis/dataframe-api/issues/212